scheduler/resource/seed_peer.go

```diff
@@ -32,7 +32,8 @@ import (
 
 const (
 	// Default value of biz tag for seed peer.
-	SeedTag = "d7y/seed"
+	SeedTag         = "d7y/seed"
+	SeedApplication = "d7y-seed"
```

Comment:
Const requires remarks. Consistent with the tag => `d7y/seed`, no need to redefine.